File: mywebroot/CS471/NOTES/EH/CODE/scope.c
#include <stdio.h>	
int z = 5;
void scope (void) {
	printf("\nz =%d",z);
	int u = z;
	printf("\nz =%d",z);
	float z = 5.5f;
	printf("\nz =%f",z);
	printf("\nu =%d\n \n",u);
}

	
int  main() {
     scope();
     return 1;

}

File: mywebroot/CS471/NOTES/RUNTIME/CODE/Scope.java
public class Scope {
static int z = 5;

	public static void main(String[] args) {
		System.out.println(z);
		int u = z;
		System.out.println(z);
		float z = 5.5f;
		System.out.println(z);
		System.out.println(u);
	}
}
